test/integration/api/users.js

```diff
@@ -240,6 +240,31 @@ describe('api: /users', () => {
                 .send({ email: 'newalice@odk.org', password: 'alice' })
                 .expect(200);
             })))));
+
+    it('should send an email to the user\'s previous email when their email changes', testService((service) =>
+      service.login('alice', (asAlice) =>
+        asAlice.get('/v1/users/current')
+          .expect(200)
+          .then((before) => asAlice.patch(`/v1/users/${before.body.id}`)
+            .send({ email: 'david123@opendatakit.org' })
+            .expect(200)
+            .then(() => {
+              const email = global.inbox.pop();
+              global.inbox.length.should.equal(0);
+              email.to.should.eql([{ address: 'alice@opendatakit.org', name: '' }]);
+              email.subject.should.equal('ODK Central account email changed');
```

Comment:
sorry, one last nit: can you also test that the appropriate values are templated into this body?

Reply:
Will do!